src/main/features/claimant-response/routes/rejection-reason.ts

```diff
@@ -36,9 +38,9 @@ export default express.Router()
         const user: User = res.locals.user
 
         draft.document.settlementAgreement = undefined
-        draft.document.formaliseRepaymentPlan = undefined
 
         draft.document.courtDetermination.rejectionReason = form.model
+        draft.document.formaliseRepaymentPlan = new FormaliseRepaymentPlan(FormaliseRepaymentPlanOption.REFER_TO_JUDGE)
```

Comment:
Is there any route test to verify this scenario?

Reply:
no , i will add one :)